src/prompts/neuro.js

```diff
@@ -42,9 +42,18 @@ export default [
   'invents new nickname',
   'whats *emote*?',
   'No~',
-  'swear word',
+  'self censors',
   'FUN AI',
   'admits to murder',
   'rants',
-  'new clone'
+  'new clone',
+  
+  //eg: "I'm from michigan"
+  'claims to not be british',
```

Comment:
She admits she is American, don't think this prompt makes much sense

Reply:
Kinda just intends to be a "wider version" of that in case she decides to say something else.
Is debatable.

Reply:
Maybe something like this?
```suggestion
  'I am <nation>',
```

Reply:
Better.